### PR TITLE
Implement cellclick event for sub cells

### PR DESCRIPTION
**Pull request by nbey**
RE: Issue #1

### CODE REVIEW

File: src/Aggregrid.js [hunk 1]
```diff
@@ -378,7 +378,7 @@ Ext.define('Jarvus.aggregrid.Aggregrid', {
             );
         }
 
-        if (target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) { // eslint-disable-line no-cond-assign
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-row')) { // eslint-disable-line no-cond-assign
```
Comment (themightychris) [inline src/Aggregrid.js:381]: @nbey this doesn't seem to accomplish anything, all cells and subcells have a `.jarvus-aggregrid-row` as a parent, and this code won't run when the RollupAggregrid's new onClick override intercepts subcell clicks

File: src/RollupAggregrid.js [hunk 2]
```diff
@@ -272,6 +272,21 @@ Ext.define('Jarvus.aggregrid.RollupAggregrid', {
...
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-subrow')) { // eslint-disable-line no-cond-assign
+            return me.onCellClick(
+                parseInt(target.up('.jarvus-aggregrid-subrow').getAttribute('data-subrow-id'), 10),
+                parseInt(target.getAttribute('data-column-id'), 10),
+                target,
+                ev
+            );
+        }
+
+        me.callParent(arguments);
```
Comment (themightychris) [inline src/RollupAggregrid.js:288]: @nbey this override interrupts the potential return value of onClick by not returning the result of me.callParent().

If you were using visual studio code with ESLint installed you'd have seen a warning about this, ensure you're using the eslint extension

File: src/RollupAggregrid.js [hunk 3]
```diff
@@ -272,6 +272,21 @@ Ext.define('Jarvus.aggregrid.RollupAggregrid', {
         me.ungroupSubRecords(staleRecords, false);
     },
 
+    onClick: function(ev, target) {
+        var me = this,
+            containerEl = me.el;
+
+        if ((target = ev.getTarget('.jarvus-aggregrid-cell', containerEl, true)) && target.up('.jarvus-aggregrid-subrow')) { // eslint-disable-line no-cond-assign
+            return me.onCellClick(
+                parseInt(target.up('.jarvus-aggregrid-subrow').getAttribute('data-subrow-id'), 10),
```
Comment (themightychris) [inline src/RollupAggregrid.js:281]: @nbey you're calling `target.up('.jarvus-aggregrid-subrow')` twice, traversing the dom can be expensive, if this result was needed twice it should have been assigned to a variable

File: src/Aggregrid.js (hunk 1 above)
Comment (nbey) [inline src/Aggregrid.js:381]: @themightychris subcells do not have a `.jarvus-aggregrid-row` as a parent, they have a `.jarvus-aggregrid-subrow` as a parent

**Pull request merged**